package.json

```diff
@@ -36,7 +36,7 @@
   },
   "dependencies": {
     "@adobe/openwhisk-loggly-wrapper": "^0.3.0",
-    "ajv": "^6.5.4",
+    "ajv": "^6.6.2",
```

Comment:
is this relevant?

Reply:
no, just a result of rebuilding. should i remove it from this PR and commit it separately?

Reply:
You can leave it in, it just might create a conflict if `master` is updated independently.